Package.resolved

```diff
@@ -6,8 +6,8 @@
         "repositoryURL": "https://github.com/apple/swift-argument-parser",
         "state": {
           "branch": null,
-          "revision": "d2930e8fcf9c33162b9fcc1d522bc975e2d4179b",
```

Comment:
These revision updates for `swift-argument-parser` and `swift-crypto` seem unrelated, did you mean to include them?

Reply:
This probably came through from a a full replacement of the Package.resolved file. I didn't change the definitions in Package.swift. Should these remain fixed?

Reply:
Yes please, I'd prefer to have these dependencies updated separately so that we can test them in isolation.

Reply:
Okay, I'll change that, one moment.

Reply:
Fixed.